### PR TITLE
add pagination into the blog&&project

### PR DESCRIPTION
**Pull request by enjoy-life0000**
added pagination boolean field into the blog&&project.

### CODE REVIEW

File: src/components/section/blog-section.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
       "type": "component",
       "repeatable": false,
       "component": "elements.section-intro"
+    },
+    "pagination": {
+      "type": "boolean"
```
Comment (Kaherdin) [inline src/components/section/blog-section.json]: Change to select with : infinite, pagination and off

**Review by Kaherdin — COMMENTED** [review]
Please make this chane

File: src/components/section/blog-section.json (hunk 1 above)
Comment (enjoy-life0000) [reply src/components/section/blog-section.json]: > Change to select with : infinite, pagination and off

1. created pagination collection have three states such like infinite, pagination and off
2. added pagination into blogsection && projectsection

Comment (Kaherdin) [reply src/components/section/blog-section.json]: Hummm, why so complicated ? I've asking for a type enum with [infinite, pagination and off].
I will check it

**Review by Kaherdin — APPROVED** [review]

**Pull request merged**